### ISSUE TITLE
Calculation Group table partition source must have Mode property set to Import

### ISSUE THREAD

**Issue by otykier** [otykier/TabularEditor#719]
Composite Models do not allow adding calc groups when the partition source Mode property is set to Default. It has to be explicitly set to Import.

This was already fixed in commit e646cf3 but no release have been created with this fix yet.

![image](https://user-images.githubusercontent.com/8976200/102864140-e9b66e00-4433-11eb-9f0d-f329e8d15a62.png)

**Comment by otykier** [comment]
As a workaround, until the above change is released, run the following script (note, when connected to Power BI Desktop, Allow Unsupported Features must be enabled under File > Preferences, for the script to work):

```csharp
foreach(var cg in Model.CalculationGroups)
{
    cg.Partitions[0].Mode = ModeType.Import;
}
```

**Comment by otykier** [comment]
Fixed in [2.14.0](https://github.com/otykier/TabularEditor/releases/tag/2.14.0).
